src/Raven.Server/Documents/Includes/Sharding/IncludeDocumentsOrchestratedSubscriptionCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Includes.Sharding;
+
+public class IncludeDocumentsOrchestratedSubscriptionCommand : AbstractIncludeDocumentsCommand
+{
+    private readonly JsonOperationContext _clusterContext;
+    private readonly CancellationToken _token;
+    private List<BlittableJsonReaderObject> _includes;
+    public override int Count => _includes?.Count ?? 0;
+
+    public IncludeDocumentsOrchestratedSubscriptionCommand(JsonOperationContext clusterContext, CancellationToken token)
+    {
+        _clusterContext = clusterContext;
+        _token = token;
+    }
+
+    public void Fill(List<BlittableJsonReaderObject> result)
+    {
+        if (_includes == null)
+            return;
+
+        foreach (var res in _includes)
+        {
+            result.Add(res);
+        }
+    }
+
+    public void Gather(List<BlittableJsonReaderObject> includes)
```

Comment:
`Fill` & `Gather` look identical to me

Reply:
`Includes` is a property now, and `Fill()` removed